### DISCUSSION TITLE
How to make a feature request

### DISCUSSION THREAD

**Discussion by jw-foss** [element-plus/element-plus#905] [Documentation]
## 👋 Welcome! We are so glad that you are here.
In this article we will cover how to make a proposal or feature request

## Principle
The principle is that we want the requirement decoupled from the business problem that you trying to solve. We do not want to implement features that could potentially bring overhead to the other users who actually do not need it. Because in the end, `Element Plus` is a atomic component library, complex components should be constructed based on these atomics.

We want as many people as we can get to participant, to discuss as `Discussion`. Debates are allowed but not fighting is not.
### How to make a proposal
As you guys can see, we've have plenty of features/components throughout this repo, and sometimes we just can not get satisfied, so we end up here to make a proposal of new feature/components

#### 1. The overall description of component
For example. I'd like to propose a new feature named Bullet, under the circumstances like listing items it will be extremely useful, clear and straight forward.

#### 2. The actual requirements

For example, for this component
-  I'd like an API named size to control the bullet size. 
-  I'd like an API named indent to control the layer of bullet.
  
#### 3. If there is any reference that we can refer to

For example: Please refer to [AntDesign](https://ant.design)

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zazzaz** [comment]
😁
